CodeLite/archive.cpp

```diff
@@ -552,12 +551,11 @@ bool Archive::Write(const wxString& name, const wxStringMap_t& strinMap)
     node->AddAttribute(wxT("Name"), name);
 
     // add an entry for each wxString in the array
-    wxStringMap_t::const_iterator iter = strinMap.begin();
-    for(; iter != strinMap.end(); ++iter) {
+    for (const auto& [key, value] : strinMap) {

```

Comment:
Lets fix the name into `stringMap`

Reply:
ok, (I even miss it and didn't add to typos to fix)

Reply:
typo fixed in whole file.